pkg/cmd/status/status.go

```diff
@@ -286,7 +286,29 @@ func (s *StatusGetter) LoadNotifications() error {
 					if !ok {
 						return nil
 					}
-					if actual, err := s.ActualMention(n.Subject.LatestCommentURL); actual != "" && err == nil {
+					actual, err := s.ActualMention(n.Subject.LatestCommentURL)
+
+					if err != nil {
+						var httpErr api.HTTPError
+						httpStatusCode := -1
+
+						if errors.As(err, &httpErr) {
+							httpStatusCode = httpErr.StatusCode
+						}
+
+						switch httpStatusCode {
+						case 403:
+							s.addAuthError(httpErr.Message, factory.SSOURL())
+							return nil
+						case 404:
+							actual = "(could not fetch comment)"
```

Comment:
Do we actually want to display anything to the user in this case? They have no actions they can take to view the comment so this message does not provide a lot of value in my opinion. I would rather see us just skip to the next one and not display anything.

Reply:
I also waffled on that. Personally, I prefer seeing that a notification exists even if I can't see the text. The comment URL still shows up, which gives some context, e.g. indicating there's a comment a former client meant to be seen. (I suspect that they only exist for comments made before access was restricted, not new comments tagged privately…but I don't know). In that case, the action might be to reach out to the client and let them know they might need to fill you in directly.

🤷  But I also don't feel too strongly about it.

Reply:
I think the more common case of this occurring is that a comment was purposefully deleted for one reason or another and in that case showing nothing seems like the correct behavior. 

For now lets go with not displaying anything and we can change the behavior in the future if we hear from users that they would rather have the other behavior.